func/python/pyc_check.py

```diff
@@ -0,0 +1,44 @@
+import bisect
+import dis
+import marshal
+import io
+
```

Comment:
For a long time I thought pyc files were the problem, so this checks they are read and parsed properly.